src/utilities/sequence-editor/from-seq-json.test.ts

```diff
@@ -284,7 +284,7 @@ C FSW_CMD_3
     const sequence = await seqJsonToSequence(JSON.stringify(seqJson));
     const expectedSequence = `@ID "42"
 
-A2024-001T00:00:00 FSW_CMD_0 TRUE 0xFF "Hello" World [FALSE 0xAA "Foo" BAR TRUE 0xBB "Baz" BAT]
+A2024-001T00:00:00 FSW_CMD_0 true 0xFF "Hello" World [false 0xAA "Foo" BAR true 0xBB "Baz" BAT]
```

Comment:
Does this mean that it no longer supports uppercase `TRUE`/`FALSE`?

Reply:
We started out using 'TRUE' and 'FALSE' for boolean values in the prototype. Later, we decided to use 'true' and 'false' instead. I didn't catch the change and kept using the old way. Clipper didn't have this problem because they use a different system (enum based) for true/false values, so the mistake went unnoticed.

Reply:
👍🏻 do you think there would be a case to support both upper and lower cases?

Reply:
We shouldn't support both uppercase and lowercase booleans. Most languages only use one or the other. To keep things simple and consistent with our grammar, which has used lowercase for a while, we should stick with lowercase. Plus, we already support lowercase booleans as used by @MODELS, so it's just aligning with what we've already done.

Feel free to ask @joswig when he gets back from vacation though. We have no need to rush this pr into develop

Reply:
I'm all for keeping it consistent with what already exists! Thanks!